Explorer/Assets/Scripts/ECS/Unity/GLTFContainer/Asset/Systems/PrepareGltfAssetLoadingSystem.cs

```diff
@@ -35,19 +37,21 @@ protected override void Update(float t)
         }
 
         [Query]
-        [None(typeof(StreamableLoadingResult<GltfContainerAsset>), typeof(GetAssetBundleIntention))]
+        [None(typeof(StreamableLoadingResult<GltfContainerAsset>), typeof(GetAssetBundleIntention), typeof(GetGLTFIntention))]
         private void Prepare(in Entity entity, ref GetGltfContainerAssetIntention intention)
         {
             // Try load from cache
-            if (cache.TryGet(intention.Hash, out GltfContainerAsset asset))
+            if (cache.TryGet(intention.Hash, out GltfContainerAsset? asset))
             {
                 // construct the result immediately
                 World.Add(entity, new StreamableLoadingResult<GltfContainerAsset>(asset));
                 return;
             }
 
-            // If not in cache, try load from asset bundle
-            World.Add(entity, GetAssetBundleIntention.Create(typeof(GameObject), $"{intention.Hash}{PlatformUtils.GetPlatform()}", intention.Name));
+            if (localSceneDevelopment)
+                World.Add(entity, GetGLTFIntention.Create(intention.Name, intention.Hash));
+            else
+                World.Add(entity, GetAssetBundleIntention.Create(typeof(GameObject), $"{intention.Hash}{PlatformUtils.GetPlatform()}", intention.Name));
```

Comment:
I accept suggestion number 1, but number 2 doesn't really apply since nothing in the `World.Add` flow can throw an exception, so I don't think it's worth enclosing it in a `try-catch` block